src/prowjobsscraper/scraper.py

```diff
@@ -33,13 +33,23 @@ def execute(self, jobs: prowjob.ProwJobs):
         # Retrieve executed steps for each job
         steps = self._step_extractor.parse_prow_jobs(jobs)
 
+        # Retrieve equinix machines usages not already stored
+        usages = usages = [
+            usage
+            for usage in self._equinix_extractor.get_project_usages()
+            if usage.job_build_id not in known_build_ids
```

Comment:
compare against the `build_id`s extracted from the `usages` index, because prow and equinix usages may not return the same `build_id`s at a given time.